specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/reservation.json

```diff
@@ -16,20 +16,17 @@
     "application/json"
   ],
   "paths": {
-    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/reservations": {
```

Comment:
This path exists in the [previous GA](https://github.com/Azure/azure-rest-api-specs/blob/18e83e24c36eed85a240103b476de0220ace7f84/specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json#L3598).

Are you sure it can be removed? I think ARM team will push back on this.

Reply:
Yeah, CI checks are failing

https://github.com/Azure/azure-rest-api-specs/pull/22321/checks?check_run_id=10897962237

> The default tag should contain all APIs. The API path /providers/Microsoft.Billing/billingAccounts/{}/billingProfiles/{}/reservations is not in the default tag. Please make sure the missing API swaggers are in the default tag.